app/models/category.rb

```diff
@@ -1,3 +1,8 @@
 class Category < ApplicationRecord
   has_many :books, dependent: :nullify
+
+  belongs_to :parent, class_name: "Category", foreign_key: "parent_id",
```

Comment:
```suggestion
  belongs_to :parent, class_name: Category.name, foreign_key: :parent_id,
```
Sửa tương tự như ở bên dưới

Reply:
Fixed